### PR TITLE
Style/addresses/refactor

### PR DESCRIPTION
**Pull request by taylorjdawson**

### CODE REVIEW

File: src/address.js [hunk 1]
```diff
@@ -204,10 +232,11 @@ Returns null if no address is found.
...
-   * @param hash - The address of the account.
-   * @param filterOptions - The filter options associated with the request.
-   * @returns The balance data of the account or if no address is found.
+   * @param {string} hash - The address of the account.
+   * @param {object} [filterOptions] - The filter options associated with the request. See [docs](https://docs.amberdata.io/reference#get-current-account-balance) for more details.
+   * @returns {Promise<object>} The balance data of the account or if no address is found.
    * @example
+   * const latestBalance = await web3data.address.getLatestBalance('0x06012c8cf97bead5deae237070f9587f8e7a266d')
    */
   getLatestBalance(hash, filterOptions = {}) {
```
Comment (TrevorJTClarke) [inline src/address.js]: @taylorjdawson this method still is weird... We have getBalance (which i thought was latest), getLatestBalance, getHistoricalBalance. How do i know which to use in my project?

File: src/address.js [hunk 2]
```diff
@@ -254,10 +285,11 @@ Returns null if no address is found.
...
    * Retrieves the latest account and token balances for the specified address.
    *
-   * @param hash - The address of the account.
-   * @param filterOptions - The filter options associated with the request.
-   * @returns The balance data of the account.
+   * @param {string} hash - The address of the account.
+   * @param {object} [filterOptions] - The filter options associated with the request. See [docs](https://docs.amberdata.io/reference#account-balances) for more details.
+   * @returns {Promise<object>} The balance data of the account.
    * @example
+   * const balances = await web3data.address.getBalances('0x06012c8cf97bead5deae237070f9587f8e7a266d')
```
Comment (TrevorJTClarke) [inline src/address.js]: @taylorjdawson another weird thing here... Why do we need this method? this is very confusing

File: src/address.js [hunk 3]
```diff
@@ -240,10 +270,11 @@ Returns null if no address is found.
...
    * Retrieves the latest account and token balances for the specified address(es).
    *
-   * @param hashes - The array or string containing the address(es) of the account.
-   * @param filterOptions - The filter options associated with the request.
-   * @returns The balance data of the account(s).
+   * @param {Array} hashes - The array or string containing the address(es) of the account.
+   * @param {object} [filterOptions] - The filter options associated with the request. See [docs](https://docs.amberdata.io/reference#account-balances) for more details.
+   * @returns {Promise<object>} The balance data of the account(s).
    * @example
+   * const balances = await web3data.address.getMultipleBalances(['0x06012c8cf97bead5deae237070f9587f8e7a266d', '0x3f5ce5fbfe3e9af3971dd833d26ba9b5c936f0be'])
```
Comment (TrevorJTClarke) [inline src/address.js]: @taylorjdawson I thought we discussed supporting multiple addresses in the "getBalance" method. This is less confusing and simplifies logic for developers

File: test/contract.test.js [hunk 4]
```diff
@@ -32,20 +32,20 @@ test('throws exception when calling getDetails without hash', async t => {
...
     }, { instanceOf: Error, message: 'No contract address supplied' });
 });
 
-/*********** Test getAudit() ***********/
-test('Successfully gets contract audit', async t => {
-    let response = await t.context.web3data.contract.getAudit(TOKEN_ADDRESS);
-    t.is(response.status, 200)
+/*********** Test getSecurityAudit() ***********/
+test('Successfully calls getSecurityAudit', async t => {
+    const audit = await t.context.web3data.contract.getAudit(TOKEN_ADDRESS);
```
Comment (TrevorJTClarke) [inline test/contract.test.js]: @taylorjdawson looks like fallback test?

**Comment by taylorjdawson** [comment]
> @taylorjdawson see comments!

These methods were added 2 months ago (see [commit](https://github.com/web3data/web3data-js/commit/5dc1b344739b13785700c333df3daff563bfdd33)). I am pretty sure that discussion has taken place since then. But either way it's out of the scope of this commit. This commit is just for docs and changing the response to not include `records`.

File: src/address.js (hunk 1 above)
Comment (taylorjdawson) [reply src/address.js]: As it stand `getBalance` get's historical or latest depending on filters.  `getLatestBalance` & `getHistoricalBalance` are the individual commands that `getBalance` uses under the hood. Not saying this is the right or preferred way but that's what it's doing.

File: src/address.js (hunk 3 above)
Comment (taylorjdawson) [reply src/address.js]: Similar to [above](https://github.com/web3data/web3data-js/pull/61#discussion_r342744009). But I think we just need a plan in place to simplify account balance retrieval across the board.

**Comment by TrevorJTClarke** [comment]
@taylorjdawson some notes for support:
Old methods: (to be deprecated)
- getLatestBalance, getHistoricalBalance, getMultipleBalances, getBalances, getBalancesBatch

NOTE: Remove support for web3.js string return for `getBalance`

Keep Methods & Use Cases:
- getBalance

A few examples:
1. Generic single address
2. Historical single address
3. Generic multiple addresses
4. Pie in the Sky multi-blockchain multiple addresses

1. Example:
```
const b = wb3.address.getBalance('0x1234f9f9f9ff9...')
console.log('b') // { balance: 12323432, tokens: [ ... ]}
```

2. Example:
```
const b = wb3.address.getBalance('0x1234f9f9f9ff9...', { startDate: +new Date() - 24 * 6, endDate: +new Date() })
console.log('b') // [{ balance: 12323432, tokens: [ ... ]}]
```

3. Example:
```
const b = wb3.address.getBalance(['0x1234f9f9f9ff9...', '0xddaaa9f9f9ff9...'])
console.log('b') // {
  '0x1234f9f9f9ff9...': { balance: 12323432, tokens: [ ... ]}
  '0xddaaa9f9f9ff9...': { balance: 999323432 }
}
```

4. Example:
```
const b = wb3.address.getBalance(['0x1234f9f9f9ff9...', 'f99991234f9f9f9ff9...'])
console.log('b') // {
  '0x1234f9f9f9ff9...': { balance: 12323432, tokens: [ ... ]} // ethereum
  '0x1234f9f9f9ff9...': { balance: 12323432, tokens: [ ... ]} // ethereum rinkeby
  'f99991234f9f9f9ff9...': { balance: 999323432 } // bitcoin or other
}
```

Lastly, this PR should remove documentation and methods that do not make sense given above examples

**Comment by TrevorJTClarke** [comment]
> @taylorjdawson some notes for support:
> Old methods: (to be deprecated)
> 
> * getLatestBalance, getHistoricalBalance, getMultipleBalances, getBalances, getBalancesBatch
> 
> NOTE: Remove support for web3.js string return for `getBalance`
> 
> Keep Methods & Use Cases:
> 
> * getBalance
> 
> A few examples:
> 
> 1. Generic single address
> 2. Historical single address
> 3. Generic multiple addresses
> 4. Pie in the Sky multi-blockchain multiple addresses
> 5. Example:
> 
> ```
> const b = wb3.address.getBalance('0x1234f9f9f9ff9...')
> console.log('b') // { balance: 12323432, tokens: [ ... ]}
> ```
> 
> 1. Example:
> 
> ```
> const b = wb3.address.getBalance('0x1234f9f9f9ff9...', { startDate: +new Date() - 24 * 6, endDate: +new Date() })
> console.log('b') // [{ balance: 12323432, tokens: [ ... ]}]
> ```
> 
> 1. Example:
> 
> ```
> const b = wb3.address.getBalance(['0x1234f9f9f9ff9...', '0xddaaa9f9f9ff9...'])
> console.log('b') // {
>   '0x1234f9f9f9ff9...': { balance: 12323432, tokens: [ ... ]}
>   '0xddaaa9f9f9ff9...': { balance: 999323432 }
> }
> ```
> 
> 1. Example:
> 
> ```
> const b = wb3.address.getBalance(['0x1234f9f9f9ff9...', 'f99991234f9f9f9ff9...'])
> console.log('b') // {
>   '0x1234f9f9f9ff9...': { balance: 12323432, tokens: [ ... ]} // ethereum
>   '0x1234f9f9f9ff9...': { balance: 12323432, tokens: [ ... ]} // ethereum rinkeby
>   'f99991234f9f9f9ff9...': { balance: 999323432 } // bitcoin or other
> }
> ```
> 
> Lastly, this PR should remove documentation and methods that do not make sense given above examples

@taylorjdawson lookin much better, thanks!

**Pull request merged**